addons/aldryn-django/aldryn_django/startup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import os
+import dotenv
+from getenv import env
+import django.core.management
+
+
+def manage(path):
+    _setup(path=path)
+    utility = django.core.management.ManagementUtility(None)
+    utility.execute()
+
+
+def wsgi(path):
+    _setup(path=path)
+    from django.core.wsgi import get_wsgi_application
+    from dj_static import Cling
+    # evil workaround for gunicorn not being able to import everything
```

Comment:
yeah, I'm going to set it up with uwsgi. But it is likely that the same hack is needed for uwsgi as well.